.buildkite/pipelines/pull_request/renovate.yml

```diff
@@ -0,0 +1,34 @@
+steps:
+  # - command: .buildkite/scripts/lifecycle/pre_build.sh
+  #   label: Pre-Build
+  #   timeout_in_minutes: 10
+  #   agents:
+  #     machineType: n2-standard-2
+
+  # - wait
+
+  # - command: .buildkite/scripts/steps/build_kibana.sh
+  #   label: Build Kibana Distribution
+  #   agents:
+  #     machineType: n2-standard-8
+  #     preemptible: true
+  #     diskSizeGb: 125
+  #   key: build
+  #   if: "build.env('KIBANA_BUILD_ID') == null || build.env('KIBANA_BUILD_ID') == ''"
+  #   timeout_in_minutes: 90
+  #   retry:
+  #     automatic:
+  #       - exit_status: '-1'
+  #         limit: 3
+
+  - command: .buildkite/scripts/steps/quick_checks.sh
```

Comment:
We can scope quick checks down to what is necessary using:

`node scripts/quick_checks.js --checks .buildkite/scripts/steps/checks/licenses.sh`

Might need a `sh` wrapper around it though, idr. Or pass args along from `quick_checks.sh` to the `js` script, and default to all checks if no args.

Reply:
I created an `sh` wrapper that only targets the renovate validation, and it leaves us room to add additional renovate checks in the future as part of this step